app/models/volunteer.rb

```diff
@@ -124,66 +129,76 @@ def has_main_region?
   end
 
   def main_region
-    self.regions[0]
-  end
-
-  def region_ids
-    self.regions.collect{ |r| r.id }
+    regions.first
```

Comment:
Possibly ;)

The `region_ids` method was removed entirely because ActiveRecord provides it for `has_many` relationships automatically. The `main_region` method implementation was changed from `self.regions[0]` to `regions.first`.